internal/eigenState/submittedDistributionRoots/submittedDistributionRoots.go

```diff
@@ -22,7 +22,7 @@ import (
 	"gorm.io/gorm/clause"
 )
 
-type SubmittedDistributionRoots struct {
+type SubmittedDistributionRoot struct {
```

Comment:
If we wanna drop the `s` let's rename the folder/file to match

Reply:
do we? for example, `avsOperators.go` centers around a `AvsOperatorStateChange` struct. the model itself is still `SubmittedDistributionRootsModel`

Reply:
Ah I see what you're thinking now. This is fine the way you have it